src/Block/BlockContextManager.php

```diff
@@ -139,7 +139,8 @@ private function configureSettings(OptionsResolver $optionsResolver, BlockInterf
                 ->addAllowedTypes('extra_cache_keys', 'array')
                 ->addAllowedTypes('attr', 'array')
                 ->addAllowedTypes('ttl', 'int')
-                ->addAllowedTypes('template', ['string', 'bool'])
+                // NEXT_MAJOR: Remove bool.
```

Comment:
IMO, we should [deprecate](https://symfony.com/doc/current/components/options_resolver.html#deprecating-the-option) the types we want to remove in the next major.